src/backend/marsha/settings.py

```diff
@@ -324,6 +351,10 @@ class Production(Base):
     }
     AWS_STATIC_BUCKET_NAME = values.Value("production-marsha-static")
 
+    # folder where static will be stored. It matches the path_pattern used
+    # in the cloudfront configuration.
+    AWS_LOCATION = Base.STATIC_URL
```

Comment:
@sampaccoud Is it ok for you ? This is working

Reply:
@lunika nice!